fabric-renderer-indigo/src/client/java/net/fabricmc/fabric/impl/client/indigo/renderer/render/ItemRenderContext.java

```diff
@@ -317,40 +288,33 @@ public void accept(BakedModel model) {
 		@Override
 		public void accept(BakedModel model, @Nullable BlockState state) {
 			if (hasTransform()) {
+				MutableQuadViewImpl editorQuad = ItemRenderContext.this.editorQuad;
+
 				// if there's a transform in effect, convert to mesh-based quads so that we can apply it
 				for (int i = 0; i <= ModelHelper.NULL_FACE_ID; i++) {
 					final Direction cullFace = ModelHelper.faceFromIndex(i);
 					random.setSeed(ITEM_RANDOM_SEED);
 					final List<BakedQuad> quads = model.getQuads(state, cullFace, random);
 					final int count = quads.size();
 
-					if (count != 0) {
-						for (int j = 0; j < count; j++) {
-							final BakedQuad q = quads.get(j);
-							editorQuad.fromVanilla(q, IndigoRenderer.MATERIAL_STANDARD, cullFace);
-							renderMeshQuad(editorQuad);
-						}
+					for (int j = 0; j < count; j++) {
+						final BakedQuad q = quads.get(j);
+						editorQuad.fromVanilla(q, IndigoRenderer.MATERIAL_STANDARD, cullFace);
+						// Call renderQuad directly instead of emit for efficiency
```

Comment:
Won't this be slower again when we will route the model through `QuadEmitter#emit`?